src/main/webapp/app/overview/visualizations/exercise-scores-chart/exercise-scores-chart.component.ts

```diff
@@ -15,18 +15,26 @@ import { round } from 'app/shared/util/utils';
     templateUrl: './exercise-scores-chart.component.html',
     styleUrls: ['./exercise-scores-chart.component.scss'],
 })
-export class ExerciseScoresChartComponent implements AfterViewInit {
+export class ExerciseScoresChartComponent implements AfterViewInit, OnChanges {
     @Input()
+    filteredExerciseIDs: number[];
+
     courseId: number;
```

Comment:
courseId now is not annotated with @Input(), is this on purpose?

Reply:
It was never used as an input, so I did not see any reason to keep it